src/docs/primitives/a-model.md

```diff
@@ -29,3 +29,9 @@ Using scale to increase to size of the loaded tree model:
 ```html
 <a-model scale="10 10 10" src="https://aframe.io/aframe/examples/_models/tree1/tree1.dae"></a-model>
 ```
```

Comment:
ah, right. and we don't allow that with primitives yet. so it should be like this:

```html
<a-entity loader="src: url(object.obj); mtl: url(object.mtl)"></a-entity>
```

great catch! I'll open a follow-up PR.